AbdoulNuru/Teamwork
# #168973734 add signup endpoint with database

### What does this PR do?

Have sign up endpoint setup with database

### Description of task to be completed?

Build sign up endpoint with Postgresql database to persist our data.

### How should this be manually tested?

- Clone this repo to your local repository using `git clone <link>`
- Install all project dependencies using `npm install <package name>`
- In your command line use, `npm run dev` to run the project
- Now using postman u can send a `post` request to this URL `/api/v2/auth/signup`
- Make sure in your request header you have *key: Content-Type value: application/json*
- And in your request body you must pass your credentials in a JSON format.

### What are the relevant pivotal tracker stories?

[#168973734](https://www.pivotaltracker.com/story/show/168973734)

### Screenshots(if appropriate)?
![Screenshot (153)](https://user-images.githubusercontent.com/53453380/66404628-1924ba80-e9e9-11e9-941c-b24a94b5cd44.png)
![Screenshot (154)](https://user-images.githubusercontent.com/53453380/66404631-19bd5100-e9e9-11e9-85e7-819ec493d611.png)
![Screenshot (155)](https://user-images.githubusercontent.com/53453380/66404635-1a55e780-e9e9-11e9-907c-1bc712005d7b.png)
![Screenshot (152)](https://user-images.githubusercontent.com/53453380/66404636-1a55e780-e9e9-11e9-896d-896267756dbc.png)
